src/posts/lucky_0.27.cr

```diff
@@ -0,0 +1,104 @@
+
+    [Lucky v0.27](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md)
+    is officially out. It's a smaller release, but brings some pretty big steps. We hope
+    you're as stoked about the progress as we are!
+
+    ### How to upgrade
+
+    See the [UPGRADE_NOTES](https://github.com/luckyframework/lucky/blob/master/UPGRADE_NOTES.md#upgrading-from-025-to-026).
+
+    Or you can view the changes using [LuckyDiff](https://luckydiff.com/?from=0.26.0&to=0.27.0).
+    This tool helps see what has changed between versions and will be included in upgrade notes from now on.
+
+    ## Here's what's new
+
+    ### Crystal 1.0 Support
+
+    For the most part, Crystal 1.0 worked with Lucky 0.26 with the exception of cookies.
+    For the Crystal 1.0 release, there were a few changes around cookies:
+    * [remove implicit path `/`](https://github.com/crystal-lang/crystal/pull/10491)
+    * [remove implicit en-/decoding of cookie values](https://github.com/crystal-lang/crystal/pull/10485)
+    * [remove cookie name decoding](https://github.com/crystal-lang/crystal/pull/10442)
+    * [split from_headers in to separate methods](https://github.com/crystal-lang/crystal/pull/10486)
+
+    These changes mean that, with Crystal 1.0, cookie values that contain certain characters like
+    `\\n`, `"`, `;`, and a few others, are no longer valid.
+
+    Prior to Lucky 0.27, Lucky session cookies were encoded using `Base64.encode` (which adds a newline character).
+    This was [patched](https://github.com/luckyframework/lucky/pull/1467) in Lucky 0.27.1.
+
+    ### Introduction of Breeze
+
+    [Breeze](https://github.com/luckyframework/breeze) is a new development dashboard system
+    for Lucky apps. It allows you to get a higher overview of each request you make in your app.
+    See which queries ran for a specific request, as well as which pipes, cookies, sessions, and headers
+    all interacted for that request.
+
+    ![Breeze](#{asset("images/lucky_breeze.png")})
+
+    Breeze also allows you to create your own extensions. It comes with a built-in extension for viewing
+    [Carbon Email](https://github.com/luckyframework/carbon) previews. This makes designing your emails
+    a lot easier, and all right in your browser.
+
+    We are still working on Breeze, but feel free to watch the repo, and even give it a shot in your app.
+    Let us know what you think!
+
+    ### LuckyTask
+
+    This was originally called `LuckyCli::Task` which existed in the [lucky_cli](https://github.com/luckyframework/lucky_cli) shard.
+    LuckyCli was required in any shard that included custom tasks. (e.g. Avram, Lucky, Breeze, etc...)
+
+    By abstracting this out to a separate shard, we gain about ~17% (or more) speed in compilation
+    time. This is mainly on cold cache, but it means your apps won't sit in the CI as long as they were!
+
+    Who doesn't love some performance boost?
```

Comment:
Technically it'd call it a "speed up" because it doesn't affect (I think) how an app performs, per se. I put this here so that if you are worried about that kind of distinction you can change it, but don't worry about it if you're not 😄